src/libraries/System.Net.Http/src/System/Net/Http/Headers/HttpRequestHeaders.cs

```diff
@@ -160,6 +160,8 @@ private T GetSpecializedCollection<T>(int slot, Func<HttpRequestHeaders, T> crea
             set { SetOrRemoveParsedValue(KnownHeaders.MaxForwards.Descriptor, value); }
         }
 
+        /// <summary>Gets or sets the value of the <see langword=":protocol" /> pseudo-header for an HTTP request.</summary>
```

Comment:
Interesting. I didn't know using such a langword would be allowed. Did you see any build failures?

cc @gewarren

Reply:
The `langword` attribute is used for other HttpRequestHeaders members, I just copied the pattern.

Reply:
Anything can be a `langword`. AFAIK it's just another way of formatting it as code.

Reply:
TIL!